scenario.py

```diff
@@ -71,10 +72,29 @@ def __init__(self, params, experiment_path):
 
         self.contributivity_list = []
 
-        if 'epoch_count' in params.keys():
+        if 'epoch_count' in params:
             self.epoch_count = params['epoch_count']
         else:
             self.epoch_count = 40
+            
+        methods_default = [
+            "Shapley values",
+            "Independant scores",
+            "TMCS values"
+        ]
+        
+        self.methods = []
+        if not 'methods' in params:
+            self.methods = methods_default
+        else:
+            if not params['methods']:
```

Comment:
Why do we need this check ? Is it the same check as in line 87 ?

Reply:
@RomainGoussault 
line 87: we check if the field `methods` is given in the config file.
line 90: if the `methods` field is given in config file, we check if it is empty or not.